modules/op-configuring-ssh-authentication-for-git.adoc

```diff
@@ -0,0 +1,97 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-configuring-ssh-authentication-for-git_{context}"]
+= Configuring SSH authentication for Git
+
+For SSH authentication, {pipelines-shortname} creates a `~/.ssh/config` file containing the SSH private key specified in the secret, before executing any steps in a task run or a pipeline run. When the steps execute, {pipelines-shortname} uses the key to retrieve the pipeline resources specified in a run.
+
+[NOTE]
+====
+It is recommended to use SSH based authentication over basic authentication.
```

Comment:
Please review the IBM SG section on "Claims and recommendations."
```suggestion
Consider using SSH-based authentication instead of basic authentication.
```